commit 06dcb0138f5f815f50142234b2e06f5540471715
Author: Ricardo Ribalda <ribalda@chromium.org>
Date:   Wed Jan 4 15:38:46 2023 +0100

    Documentation: sysctl: correct kexec_load_disabled
    
    Patch series "kexec: Add new parameter to limit the access to kexec", v6.
    
    Add two parameter to specify how many times a kexec kernel can be loaded.
    
    These parameter allow hardening the system.
    
    While we are at it, fix a documentation issue and refactor some code.
    
    
    This patch (of 3):
    
    kexec_load_disabled affects both ``kexec_load`` and ``kexec_file_load``
    syscalls. Make it explicit.
    
    Link: https://lkml.kernel.org/r/20221114-disable-kexec-reset-v6-0-6a8531a09b9a@chromium.org
    Link: https://lkml.kernel.org/r/20221114-disable-kexec-reset-v6-1-6a8531a09b9a@chromium.org
    Signed-off-by: Ricardo Ribalda <ribalda@chromium.org>
    Reviewed-by: Steven Rostedt (Google) <rostedt@goodmis.org>
    Acked-by: Baoquan He <bhe@redhat.com>
    Cc: Bagas Sanjaya <bagasdotme@gmail.com>
    Cc: "Eric W. Biederman" <ebiederm@xmission.com>
    Cc: Guilherme G. Piccoli <gpiccoli@igalia.com>
    Cc: Joel Fernandes (Google) <joel@joelfernandes.org>
    Cc: Jonathan Corbet <corbet@lwn.net>
    Cc: Philipp Rudo <prudo@redhat.com>
    Cc: Ross Zwisler <zwisler@kernel.org>
    Cc: Sergey Senozhatsky <senozhatsky@chromium.org>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>

diff --git a/Documentation/admin-guide/sysctl/kernel.rst b/Documentation/admin-guide/sysctl/kernel.rst
index 46e3d62c0eea..b53c0235cb43 100644
--- a/Documentation/admin-guide/sysctl/kernel.rst
+++ b/Documentation/admin-guide/sysctl/kernel.rst
@@ -453,9 +453,10 @@ this allows system administrators to override the
 kexec_load_disabled
 ===================
 
-A toggle indicating if the ``kexec_load`` syscall has been disabled.
-This value defaults to 0 (false: ``kexec_load`` enabled), but can be
-set to 1 (true: ``kexec_load`` disabled).
+A toggle indicating if the syscalls ``kexec_load`` and
+``kexec_file_load`` have been disabled.
+This value defaults to 0 (false: ``kexec_*load`` enabled), but can be
+set to 1 (true: ``kexec_*load`` disabled).
 Once true, kexec can no longer be used, and the toggle cannot be set
 back to false.
 This allows a kexec image to be loaded before disabling the syscall,